tests/src/core/testqgsarcgisrestutils.cpp

```diff
@@ -253,6 +253,69 @@ void TestQgsArcGisRestUtils::testParseMarkerSymbol()
   QCOMPARE( markerLayer->strokeWidth(), 5.0 );
   QCOMPARE( markerLayer->strokeWidthUnit(), Qgis::RenderUnit::Points );
 
+  // esriTS
+  const QVariantMap fontMap = jsonStringToMap( "{"
+                              "\"type\": \"esriTS\","
+                              "\"text\": \"text\","
+                              "\"color\": ["
+                              "78,"
+                              "78,"
+                              "78,"
+                              "255"
+                              "],"
+                              "\"backgroundColor\": ["
+                              "0,"
+                              "0,"
+                              "0,"
+                              "0"
+                              "],"
+                              "\"borderLineSize\": 2,"
+                              "\"borderLineColor\": ["
+                              "255,"
+                              "0,"
+                              "255,"
+                              "255"
+                              "],"
+                              "\"haloSize\": 2,"
+                              "\"haloColor\": ["
+                              "0,"
+                              "255,"
+                              "0,"
+                              "255"
+                              "],"
+                              "\"verticalAlignment\": \"bottom\","
+                              "\"horizontalAlignment\": \"left\","
+                              "\"rightToLeft\": false,"
+                              "\"angle\": 0,"
```

Comment:
```suggestion
                              "\"angle\": 45,"
```

That'll allow us to check the angle conversion logic too